src/en/howto-privatecloud.md

```diff
@@ -17,14 +17,23 @@ required by the end user. However, those setting up a private
 cloud, or who want to change how things work (eg use a different
 Ubuntu image), can create their own metadata.
 
-This page explains how to use Juju and additional tools to generate
-this Simplestreams metadata and configure OpenStack to use them.
+There are a few ways to accomplish this based on the OpenStack
+configuration and your level of permissions with juju and/or the
+OpenStack deployment.
 
+* If you are a general user start with [Create image metadata with Juju][general].
+* If you have admin or operator permissions in the OpenStack deployment, start
+with [Create image metadata with Juju][general] and continue with
+[Upload the Simplestreams Metadata to Swift][swift].
+* If the OpenStack deployment was done with juju and you are permissions, use the [glance-simplestreams-sync charm][gsscharm].
```

Comment:
Maybe "If the OpenStack deployment was done with juju and you have permissions to deploy charms alongside the OpenStack charms then"